src/Spark.Web/Services/ServerMetadata.cs

```diff
@@ -23,7 +23,8 @@ public List<ResourceStat> GetResourceStats()
 
 			foreach (string name in names)
 			{
-				var search = _searchService.GetSnapshot(name,new SearchParams { Summary = SummaryType.Count }); 
+				// synchronous for index.cshtml
+				var search = _searchService.GetSnapshot(name,new SearchParams { Summary = SummaryType.Count }).GetAwaiter().GetResult(); 
```

Comment:
Can use Task.WhenAll() here to run them in parallel.

Reply:
This function is not asynchronous therefore you cannot.